packages/flutter/lib/src/material/floating_action_button_theme.dart

```diff
@@ -207,6 +217,7 @@ class FloatingActionButtonThemeData with Diagnosticable {
       extendedIconLabelSpacing: lerpDouble(a?.extendedIconLabelSpacing, b?.extendedIconLabelSpacing, t),
       extendedPadding: EdgeInsetsGeometry.lerp(a?.extendedPadding, b?.extendedPadding, t),
       extendedTextStyle: TextStyle.lerp(a?.extendedTextStyle, b?.extendedTextStyle, t),
+      mouseCursor: t < 0.5 ? a?.mouseCursor : b?.mouseCursor,
```

Comment:
ideally this needs to be tested but `FloatingActionButtonThemeData.lerp` itself  is not tested :(